### ISSUE TITLE
FileInput is not adjusting to Grid

### ISSUE THREAD

**Issue by remyvdwereld** [Amsterdam/design-system#1733]
# Describe the issue

The FileInput component is not adjusting to the Grid. I need to add a width of 100% to achieve this.

Check the difference in this example with a TextInput.

[Sandbox](https://codesandbox.io/p/sandbox/fileinput-grid-szrwns)

**Comment by RubenSibon** [comment]
Thanks for submitting the report! The Sandbox link does not seem to work (for me at least). Can you provide steps to reproduce this issue as an alternative as well?

> is it possible to remove the default padding-left in a grid? Or create a property for this?

Can you make a separate issue for this? I know it's cumbersome, but would keep the scope of the issues more clear. You could point to the same Sandbox, though.

**Comment by lattaai1** [comment]
Hello, I've fixed this issue!

This is what I've done:
I've updated the FileInput component to have a width of 100% to match the behavior of the TextInput component. This change will ensure that the FileInput adjusts properly to the Grid layout. Here's what I did:

1. Modified the `file-input.scss` file to add `inline-size: 100%;` to the `.ams-file-input` class.
2. Removed the `max-inline-size` property from the `.ams-file-input` class, as it's no longer needed.

These changes will make the FileInput component behave similarly to the TextInput component in terms of width adjustment within the Grid.

I also created pull request: https://github.com/Amsterdam/design-system/pull/1735


> [!CAUTION]  
> Disclaimer: This fix was created by Latta AI and you should never merge before you check the correctness of generated code!

---
This bug was fixed for free by Latta AI - https://latta.ai/ourmission

**Comment by RubenSibon** [comment]
Thanks, but no thanks @lattaai1.

**Comment by dlnr** [comment]
If this is a requirement for your form you could add some CSS:
```css
input[type="file"] {
  inline-size: 100%;
}
```
Until we add this to the component of course.

**Comment by VincentSmedinga** [comment]
I agree that File Input should be 100% wide, just like our comparable input components; at least until we have more advice on sizing and arranging form fields. This creates a common baseline and lets the fields better adapt to local layout styles. I’ve made a PR to implement this and will update this issue as soon as that has been released.

Note that the suggestion above requires `box-sizing: border-box;` and best uses `.ams-file-input` as the selector.

**Comment by VincentSmedinga** [comment]
@remyvdwereld This has been [resolved in release 0.14](https://github.com/Amsterdam/design-system/blob/design-system-css-v0.14.0/packages/css/src/components/file-input/file-input.scss#L22). Can you confirm?

**Comment by remyvdwereld** [comment]
LGTM!

**Comment by VincentSmedinga** [comment]
Cool, thanks!
